Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -147,6 +149,22 @@ public extension VisualInstruction.Component {
             return scale
         }
     }
+
+    struct GuidanceViewImageRepresentation: Equatable {
```

Comment:
ImageRepresentation isn’t meant to be limited to the `image` case; it just means that it contains the details for rendering an image. So does this struct, but the difference is in how the URL is interpreted. A separate struct is fine, but we should probably rename the two structs to better reflect the distinction. Perhaps one would be ScalableImageRepresentation or something to that effect. Needs more thought.